all_nodes.json

```diff
@@ -0,0 +1 @@
+{"nodes": [{"node_id": "e12afca2cc0c4d6fb2a27119010f1e20", "parents": ["a0378f4f37c5492091918f53b4da15a5", "c592ff8d473a4924bcd5655ae1c6ffd5"]}, {"node_id": "C90A70BD-23D0-4E8A-BA61-518745390EA5", "parents": ["e12afca2cc0c4d6fb2a27119010f1e20"]}, {"node_id": "D483616F-2A54-497E-96B1-1DE919677000", "parents": ["e12afca2cc0c4d6fb2a27119010f1e20"]}, {"node_id": "final", "parents": ["C90A70BD-23D0-4E8A-BA61-518745390EA5", "D483616F-2A54-497E-96B1-1DE919677000"]}, {"node_id": "a0378f4f37c5492091918f53b4da15a5", "parents": []}, {"node_id": "c592ff8d473a4924bcd5655ae1c6ffd5", "parents": []}]}
```

Comment:
it's bad practice to commit your test data to the central repository
you can keep int to test